simgui-ds.json

```diff
@@ -96,8 +96,7 @@
   ],
   "robotJoysticks": [
     {
-      "guid": "78696e70757401000000000000000000",
-      "useGamepad": true
+      "guid": "Keyboard0"
```

Comment:
is this intentional?

Reply:
these files should probably be added to the gitignore. I think ill let it slip through this pr and pr it in in another change

Reply:
cool